setup.py

```diff
@@ -12,5 +17,6 @@
   author='balloob, ggravlingen',
   author_email='no@email.com',
   url='https://github.com/ggravlingen/pytradfri',
-  download_url=DOWNLOAD_URL
+  download_url=DOWNLOAD_URL,
+  install_requires=REQUIRES,
```

Comment:
Yup, this was something on my list to Google.

I was also considering moving the aiocoap imports to the func, as they'll be imported as part of the __main__.py loading.